config/package-bundle/config/deployment.yml

```diff
@@ -33,3 +41,8 @@ spec:
           capabilities:
             drop:
             - all
+      #@ if/end data.values.deployment.nodeSelector != None:
+      nodeSelector:
+        #@ for/end key in data.values.deployment.nodeSelector:
+        #@yaml/text-templated-strings
+        (@= key @): #@ data.values.deployment.nodeSelector[key]
```

Comment:
is this just `nodeSelector: #@ data.values.deployment.nodeSelector`

Reply:
Yeh, actually I am wondering why there was this map situation in TCE / TKG ...